system/lib/libc/musl/src/thread/pthread_cond_timedwait.c

```diff
@@ -187,10 +187,6 @@ int __pthread_cond_timedwait(pthread_cond_t *restrict c, pthread_mutex_t *restri
 		__pthread_setcancelstate(PTHREAD_CANCEL_DISABLE, 0);
 	}
 
-#ifdef __EMSCRIPTEN__
-	pthread_testcancel();
```

Comment:
Is this needed, or just generally part of a cleanup?  If it not needed as part of this change can you split it out into its own change?

Reply:
See https://github.com/emscripten-core/emscripten/pull/15603#discussion_r755862451 for the reason for this change, I just moved this to PR #15657.